server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -143,6 +143,27 @@ export class RwcdsFormService {
       &$expand=dcp_ZoningResolution
     `);
 
+    // dcp_includezoningtextamendment is a field on the rwcds-form that should be "Yes" (717170000) if...
+    // there exists a "ZR" action and that action's dcp_ZoningResolution.dcp_zoningresolution is "AppendixF"
+    const zrAppendixF = projectActions.filter(action =>
```

Comment:
is `zrAppendixF` at most only one element?

Reply:
If so, I think we can use something like `projectActions.find()`, or maybe (less appropriate) `projectActions.some()` 

I got a bit confused at first because the `filter()` function makes me expect `zrAppendixF` to be an array of multiple elements.

Reply:
okay good point, I can update